src/instructlab/config/init.py

```diff
@@ -429,7 +429,7 @@ def get_params(
         guide_text = "  This guide will help you to setup your environment"
         separator = get_separator(guide_text)
         click.echo(
-            f"\n{separator}\n         Welcome to the InstructLab CLI\n{guide_text}\n{separator}\n"
+            f"\n{separator}\n         Welcome to the InstructLab Core\n{guide_text}\n{separator}\n"
```

Comment:
I think we're fine to leave CLI here, since this is an entrypoint to a CLI

Reply:
keep it, not change